src/main/scala/com/sky/kafka/configurator/Main.scala

```diff
@@ -28,48 +27,35 @@ object Main extends LazyLogging {
       .text("Session and connection timeout for Zookeeper")
   }
 
-  def main(args: Array[String]): Unit = {
+  def main(args: Array[String]) {
     logger.info(s"Running ${BuildInfo.name} ${BuildInfo.version} with args: ${args.mkString(", ")}")
     run(args) match {
-      case Success(_) =>
-        System.exit(0)
-      case Failure(_) =>
+      case Success((errors, infoLogs)) =>
+        errors.foreach(e => logger.warn(s"${e.getMessage}. Cause: ${e.getCause.getMessage}"))
+        infoLogs.foreach(msg => logger.info(msg))
+        if (errors.isEmpty) System.exit(0) else System.exit(1)
+      case Failure(t) =>
+        logger.warn(t.getMessage)
```

Comment:
@richardlundsky @lacarvalho91 After the fact I know, but shouldn't this be a `logger.error`?